tetrateci/test_1.8.sh

```diff
@@ -27,28 +32,28 @@ go test -count=1 -tags=integ ./tests/integration/security/mtlsk8sca/...  -p 1 -t
-else
-  go test -count=1 -tags=integ -timeout 30m ./tests/integration/security/.  -p 1 -test.v
+  go test -count=1 -tags=integ ./tests/integration/pilot/cni/... -p 1 -test.v
 
-  if [[ ${CLUSTER} == "aks" ]]; then
-    go test -count=1 -tags=integ -timeout 30m -run='TestStatsFilter|TestStatsTCPFilter|TestSetup|TestIstioctlMetrics|TestTcpMetric|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics' ./tests/integration/telemetry/stats/...  -p 1 -test.v
-  else
-    go test -count=1 -tags=integ ./tests/integration/telemetry/stackdriver/... -run='TestStackdriverHTTPAuditLogging|TestVMTelemetry'  -p 1 -test.v
-    go test -count=1 -tags=integ -timeout 30m ./tests/integration/telemetry/stats/...  -p 1 -test.v
-  fi
+  go test -count=1 -tags=integ -timeout 30m ./tests/integration/telemetry/stats/... -run='TestStatsFilter|TestSetup|TestIstioctlMetrics|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics'  -p 1 -test.v
 fi
 
-go test -count=1 -tags=integ ./tests/integration/security/sds_ingress/.  -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/sds_ingress_gateway/.  -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/sds_ingress_k8sca/.  -p 1 -test.v
-go test -count=1 -tags=integ -timeout 30m ./tests/integration/pilot/ -run='TestAddToAndRemoveFromMesh|TestAllNamespaces|TestAuthZCheck|TestDescribe|TestDirectoryWithRecursion|TestDirectoryWithoutRecursion|TestEmptyCluster|TestEnsureNoMissingCRDs|TestErrorLine|TestFileAndKubeCombined|TestFileOnly|TestGateway|TestIngress|TestInvalidFileError|TestIstioctlMetrics|TestJsonInputFile|TestJsonOutput|TestKubeOnly|TestLocality|TestMirroring|TestMirroringExternalService|TestProxyConfig|TestProxyStatus|TestStatsFilter|TestTcpMetric|TestTraffic|TestValidation|TestVersion|TestWASMTcpMetric|TestWait|TestWasmStatsFilter|TestWebhook' -istio.test.skipVM true  -p 1 -test.v
+if [[ ${CLUSTER} == "gke" ]]; then
+  go test -count=1 -tags=integ ./tests/integration/pilot/cni/... ${CLUSTERFLAGS}  -p 1 -test.v
 
+  go test -count=1 -tags=integ -timeout 30m ./tests/integration/security/.  -p 1 -test.v
 
-if [[ ${CLUSTER} != "aks" ]]; then
-  go test -count=1 -tags=integ ./tests/integration/pilot/cni/... ${CLUSTERFLAGS}  -p 1 -test.v
+  go test -count=1 -tags=integ ./tests/integration/telemetry/stackdriver/... -run='TestStackdriverHTTPAuditLogging|TestVMTelemetry'  -p 1 -test.v
+  go test -count=1 -tags=integ -timeout 30m ./tests/integration/telemetry/stats/...  -p 1 -test.v
+fi
+
+if [[ ${CLUSTER} == "aks" ]]; then
+  go test -count=1 -tags=integ -timeout 30m ./tests/integration/security/.  -p 1 -test.v
+  
+  go test -count=1 -tags=integ -timeout 30m -run='TestStatsFilter|TestStatsTCPFilter|TestSetup|TestIstioctlMetrics|TestTcpMetric|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics' ./tests/integration/telemetry/stats/...  -p 1 -test.v
```

Comment:
Like the above, can we place the test package first and then the run=regex? That way it is easier to read and figure out the difference.

Reply:
Sure